packages/svelte/src/reactivity/set.js

```diff
@@ -117,7 +117,11 @@ export class ReactiveSet extends Set {
 	}
 
 	clear() {
-		if (super.size !== 0) {
+		var size = super.size;
+		// Clear first, so we get nice console.log outputs with $inspect
+		super.clear();
+
+		if (size !== 0) {
```

Comment:
should we early return instead?

```suggestion
		if (super.size === 0) return
```